samples/nrf9160/nrf_cloud_mqtt_multi_service/child_image/mcuboot/boards/nrf9160dk_nrf9160_0_14_0.overlay

```diff
@@ -10,6 +10,11 @@
 	};
 };
 
+/* External flash device is disabled by default */
```

Comment:
Just checking -- is it necessary to modify mcuboot's handling of external flash?

Reply:
if you want it to be enabled, yes.

Reply:
Yep, it is -- I actually just finished confirming that it really is necessary. External-flash-partition builds fail without it